core/tests/snapshots.rs

```diff
@@ -277,7 +277,12 @@ fn run_bank_forks_snapshot_n<F>(
     )
     .unwrap();
     solana_runtime::serde_snapshot::reserialize_bank_with_new_accounts_hash(
-        accounts_package.snapshot_links.path(),
+        accounts_package
```

Comment:
this sequence appears at least 4 times. And they all contain an unwrap. can you do an impl method on accounts_package with a name that says that it expects Some?